pyproject.toml

```diff
@@ -92,6 +92,7 @@ lint = [
 test = [
     "pytest>=6.0",
     "html5lib",
+    "defusedxml>=0.7", # for security safe XML/HTML parsing
```

Comment:
Nitpick: maybe begin from `0.7.1`?  That version fixes a regression, with little else changed:

https://github.com/tiran/defusedxml/compare/v0.7.0...v0.7.1